target_postgres/sql_base.py

```diff
@@ -214,38 +332,212 @@ def get_table_schema(self, connection, name):
 
     def update_table_schema(self, connection, remote_table_json_schema, table_json_schema, metadata):
         """
-
+        Update the remote table schema based on the merged difference between
+        `remote_table_json_schema` and `table_json_schema`.
         :param connection: remote connection, type left to be determined by implementing class
         :param remote_table_json_schema: get_table_schema
         :param table_json_schema: updates for get_table_schema
-        :param metadata: additional metadata needed to implementing class
+        :param metadata: additional metadata needed by implementing class
         :return: updated_remote_table_json_schema
         """
         raise NotImplementedError('`update_table_schema` not implemented.')
```

Comment:
I think it may make sense to start moving these towards the top of the class and eventually make it more obvious that these are the methods to override. Documentation helps too of course